src/System.Devices.Gpio/publish_upload.txt

```diff
@@ -0,0 +1,3 @@
+cd ./Desktop
+put publish.zip
```

Comment:
Did you intend to commit the publish.zip to the repo?

Reply:
please don't check in binaries / zips to git 😄

Reply:
No, I don't have the intention to commit the zip file to the repo.